### PR TITLE
#21 - Added support to landscape view

### PR DESCRIPTION
**Pull request by Azooz2014**

### CODE REVIEW

**Comment by Swati4star** [comment]
Can you please squash these 2 commits into a single one?

**Comment by Azooz2014** [comment]
how can i do that ?

**Comment by Swati4star** [comment]
You can check out this [link](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -52,4 +52,4 @@
         app:headerLayout="@layout/nav_header_main"
         app:menu="@menu/activity_main_drawer" />
 
-</android.support.v4.widget.DrawerLayout>
+</android.support.v4.widget.DrawerLayout>
```
Comment (Swati4star) [inline app/src/main/res/layout/activity_main.xml]: there is no change in this file. Remove this file from here.

Comment (Azooz2014) [reply app/src/main/res/layout/activity_main.xml]: how ?

**Comment by Azooz2014** [comment]
Squashed all commits into single one , is all good?

**Comment by Swati4star** [comment]
The code looks good. I will test it & merge it.

**Comment by Swati4star** [comment]
There are few errors I noticed,

1. If you keep your device on horizontal mode initially and then open the app, the screen looks like this,
device-2017-10-19-004744

2. If I click on 'View Files', the screen looks like this.
device-2017-10-19-004909


3. When I click on 'Create PDF', it prompts me to write name of pdf. But then, it shows the dialog again. and then the app crashes.

**Comment by Azooz2014** [comment]
Cool I will start working on fixing those issues in the morning , for the 2nd bug i may remove View files from the navigation drawer in landscape layout is that ok with you?

**Comment by Swati4star** [comment]
Yes. Remove the options from navigation drawer then.

**Comment by Azooz2014** [comment]
Fixed all the above bugs you have mentioned , but i didn't encounter the last bug that caused the app to crash at all so i don't know what's that about.

**Comment by Swati4star** [comment]
I am getting the third error.

Here is the log:

> 10-20 16:02:13.333 2909-2909/swati4star.createpdf E/AndroidRuntime: FATAL EXCEPTION: main
                                                                    Process: swati4star.createpdf, PID: 2909
                                                                    java.lang.NullPointerException: Attempt to invoke virtual method 'android.os.IBinder android.view.View.getWindowToken()' on a null object reference
                                                                        at com.afollestad.materialdialogs.util.DialogUtils.hideKeyboard(DialogUtils.java:226)
                                                                        at com.afollestad.materialdialogs.MaterialDialog.dismiss(MaterialDialog.java:1810)
                                                                        at com.afollestad.materialdialogs.MaterialDialog.onClick(MaterialDialog.java:343)
                                                                        at android.view.View.performClick(View.java:5637)
                                                                        at android.view.View$PerformClick.run(View.java:22429)
                                                                        at android.os.Handler.handleCallback(Handler.java:751)
                                                                        at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                        at android.os.Looper.loop(Looper.java:154)
                                                                        at android.app.ActivityThread.main(ActivityThread.java:6119)
                                                                        at java.lang.reflect.Method.invoke(Native Method)
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)



Maybe some android version problem.
Here are the specs of my virtual device on which I am getting the above error.

screen shot 2017-10-20 at 4 06 00 pm

**Comment by Azooz2014** [comment]
Maybe its because of the virtual device , from the log i don't see anything related to my newly added methods or views , try and run it on a real device .

**Comment by Swati4star** [comment]
Thanks 👍 
Merged 👍

**Comment by Azooz2014** [comment]
🎉😄

**Pull request merged**